packages/jellyfish-api-core/__tests__/category/masternode/getMasternode.test.ts

```diff
@@ -0,0 +1,58 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { MasternodeState } from '../../../src/category/masternode'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should get a masternode', async () => {
+    let id: string = ''
+
+    const address = await client.wallet.getNewAddress()
+    await client.masternode.createMasternode(address)
+
+    await container.generate(1)
+
+    const masterNodes = await client.masternode.listMasternodes()
+    for (const mnId in masterNodes) {
+      if (masterNodes[mnId].ownerAuthAddress === address) {
+        id = mnId
+      }
+    }
+
+    const masternode = await client.masternode.getMasternode(id)
+
+    for (const masternodeKey in masternode) {
```

Comment:
Same here, empty `masternode` array still pass test.

Reply:
Added test to check for the length